torchrl/data/tensordict/utils.py

```diff
@@ -82,3 +82,52 @@ def _getitem_batch_size(
     list_iter_bs = list(iter_bs)
     bs += list_iter_bs
     return torch.Size(bs)
+
+
+def convert_ellipsis_to_idx(idx: tuple, batch_size: list):
```

Comment:
what about using `tuple` vs `typing.Tuple` ? I think the tests check for Python 3.8 as well so we should be compatible to that

Reply:
Yes we should stick to `typing` whenever possible. For now we don't check typing so those constraints are quite loose -- I'm sure there are a few `x: dict` or untyped functions here and there in the repo 😅

Reply:
Changed to Tuple and List from typing library